resourceDescriptors.yaml

```diff
@@ -235,6 +235,12 @@
   - name: gene/MODinteractions
     url: https://www.yeastgenome.org/locus/[%s]/interaction#network
 
+- db_prefix:ECO
```

Comment:
Probably doesn't matter, but it would be nice to put "ECO" in with other "External resources".